platform-version-prereq.cmake

```diff
@@ -1,22 +0,0 @@
-###
```

Comment:
But the user is still able to point to an independent HPCC-Platform clone via HPCC_SOURCE_DIR.